### PR TITLE
Update with service health initiative

### PR DESCRIPTION
**Pull request by jfaurskov**
Modify workflows to deploy service health initiative and assign it
Create Service health inititative including service health alerts and alert processing policy
Modify existing initiatives to remove service health alert policies
Update existing assign bicep files to remove alert processing policy assignment
Add assign bicep file for service health alert initiative
Modify existing parameter files to remove service health alerts
create parameter file for service health alert initiative
Update deployment guide with guidance on how to deploy service health initiative
Update alert details with more specific heading
Update policyinitiatives with correct mapping for policies to initiatives

### CODE REVIEW

**Review by arjenhuitema — DISMISSED** [review]
LGTM

File: docs/wiki/DeploymentGuide.md [hunk 1]
```diff
@@ -69,14 +70,13 @@ We recognize that Azure allows for flexibility and choice, and you may not be al
 * A management group structure that is not aligned to ALZ. Where you may only have a Platform management group without the sub management groups like Identity/ Management/ Connectivity. 
 * No management group structure.
 
-> **NOTE:** If you are looking to align your Azure environment to Azure landing zone, please see [Transition existing Azure environments to the Azure landing zone conceptual architecture](http://aka.ms/alz/brownfield).
```
Comment (paulgrimley) [inline docs/wiki/DeploymentGuide.md:72]: @jfaurskov you need to put this back in as you have removed brownfield guidance i added

**Review by paulgrimley — CHANGES_REQUESTED** [review]

File: docs/wiki/DeploymentGuide.md (hunk 1 above)
Comment (paulgrimley) [reply docs/wiki/DeploymentGuide.md:72]: @jfaurskov i need to spend sometime with you reviewing this as you changed a lot in here with removal of commands and wanted to check

**Review by paulgrimley — APPROVED** [review]
Reviewed deployment guide offline and good with changes

**Pull request merged**